packages/perseus-editor/src/editor-page.tsx

```diff
@@ -108,7 +109,6 @@ class EditorPage extends React.Component<Props, State> {
         // this.isMounted() but is still considered an anti-pattern.
         this._isMounted = true;
 
-        this.rendererMountNode = document.createElement("div");
```

Comment:
This is never used after it is created and since it is never assigned an id or class, nor added to the DOM, I can't imagine how it's used!